inc/sainexthop.h

```diff
@@ -45,11 +45,64 @@ typedef enum _sai_next_hop_type_t
     SAI_NEXT_HOP_TYPE_MPLS,
 
     /** Tunnel next hop */
-    SAI_NEXT_HOP_TYPE_TUNNEL_ENCAP
+    SAI_NEXT_HOP_TYPE_TUNNEL_ENCAP,
+
+    /** IPv6 Segment Route SID List */
+    SAI_NEXT_HOP_TYPE_SEGMENTROUTE_SIDLIST,
+
+    /** IPv6 Segment Route Endpoint Function */
+    SAI_NEXT_HOP_TYPE_SEGMENTROUTE_ENDPOINT
 
 } sai_next_hop_type_t;
 
 /**
+ * @brief Enum defining Endpoint Action types
```

Comment:
If not already done (don't think i've seen it), could you please write a comment about which endpoint-types are you supporting with this initial proposal?